usaspending_api/awards/management/commands/loadcontracts.py

```diff
@@ -26,25 +27,227 @@ def handle(self, *args, **options):
+            "minority_owned_business": self.parse_first_character(row['minorityownedbusinessflag']),
+            "women_owned_small_business": self.parse_first_character(row['iswomenownedsmallbusiness']),
+            "economically_disadvantaged_women_owned_small_business": self.parse_first_character(row['isecondisadvwomenownedsmallbusiness']),
+            "joint_venture_women_owned_small_business": self.parse_first_character(row['isjointventureecondisadvwomenownedsmallbusiness']),
+            "joint_venture_economic_disadvantaged_women_owned_small_bus": self.parse_first_character(row['isjointventureecondisadvwomenownedsmallbusiness']),
+            "veteran_owned_business": self.parse_first_character(row['veteranownedflag']),
+            "airport_authority": self.parse_first_character(row['isairportauthority']),
+            "housing_authorities_public_tribal": self.parse_first_character(row['ishousingauthoritiespublicortribal']),
+            "interstate_entity": self.parse_first_character(row['isinterstateentity']),
+            "planning_commission": self.parse_first_character(row['isplanningcommission']),
+            "port_authority": self.parse_first_character(row['isportauthority']),
+            "transit_authority": self.parse_first_character(row['istransitauthority']),
+            "foreign_owned_and_located": self.parse_first_character(row['isforeignownedandlocated']),
+            "american_indian_owned_business": self.parse_first_character(row['aiobflag']),
+            "alaskan_native_owned_corporation_or_firm": self.parse_first_character(row['isalaskannativeownedcorporationorfirm']),
+            "indian_tribe_federally_recognized": self.parse_first_character(row['isindiantribe']),
+            "native_hawaiian_owned_business": self.parse_first_character(row['isnativehawaiianownedorganizationorfirm']),
+            "tribally_owned_business": self.parse_first_character(row['istriballyownedfirm']),
+            "asian_pacific_american_owned_business": self.parse_first_character(row['apaobflag']),
+            "black_american_owned_business": self.parse_first_character(row['baobflag']),
+            "hispanic_american_owned_business": self.parse_first_character(row['baobflag']),
+            "native_american_owned_business": self.parse_first_character(row['naobflag']),
+            "subcontinent_asian_asian_indian_american_owned_business": self.parse_first_character(row['saaobflag']),
+            "us_local_government": self.parse_first_character(row['islocalgovernmentowned']),
+            "division_name": self.parse_first_character(row['divisionname']),
+            "division_number": self.parse_first_character(row['divisionnumberorofficecode'])
+        }
+
+        le = LegalEntity.objects.filter(recipient_unique_id=row['dunsnumber']).first()
+        if not le:
+            le = LegalEntity.objects.create(**recipient_dict)
+
+        return le
+
+    def parse_first_character(self, flag):
+        if len(flag) > 0:
```

Comment:
I'm sure you've thought this through, but I assume that this will cause the field in question to be set to `None` (and thus `NULL` in the db) if `len(flag) > 0` is not true. If that's the way it's supposed to work, what do you think about being explicit and returning `None` in that case (for ease of reading)?